great_expectations/render/renderer/slack_renderer.py

```diff
@@ -91,13 +91,13 @@ def render(self, validation_result=None):
         if custom_blocks:
             query["blocks"].append(custom_blocks)
 
-        documentation_url = "https://docs.greatexpectations.io/en/latest/features/validation.html#reviewing-validation-results"
+        documentation_url = "https://docs.greatexpectations.io/en/latest/tutorials/getting_started/set_up_data_docs.html#getting-started-set-up-auto-docs"
```

Comment:
This URL has both a reference to ``set_up_data_docs`` and ``set_up_auto_docs`` I think that "auto docs" here is a mistake introduced during the big docs refactor; can you update the link to be to data docs (including the target)?

Reply:
Will do